apps/src/applab/applab.js

```diff
@@ -960,6 +962,40 @@ Applab.render = function () {
     Applab.reactMountPoint_);
 };
 
+Applab.exportApp = function() {
+  var code = studioApp.editor.getValue();
+  // TODO: find another way to get this element that doesn't rely on global element id.
+  var appElement = document.getElementById('divApplab').cloneNode(true);
+  appElement.style.display = 'block';
+  appElement.classList.remove('notRunning');
+  appElement.classList.remove('withCrosshair');
+  var htmlBody = appElement.outerHTML;
+  var origin = window.location.origin;
+  var html = `
+<html>
+  <head>
+    <title>My App</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="${origin}/assets/js/en_us/common_locale.js"></script>
+    <script src="${origin}/assets/js/en_us/applab_locale.js"></script>
+    <script src="${origin}/assets/js/applab-api.js"></script>
+    <link rel="stylesheet" href="${origin}/assets/css/applab.css">
```

Comment:
I think this is TBD. I can see pros and cons for both options. I think in general it would be cool for code.org to host it's API. But to do that "right", it should really be versioned, which it currently isn't. So either we just link to code.org assets and start versioning them, or we bundle the assets into the zip file. I suspect the latter will be easier to support long term. @poorvasingal did you have an opinion about this?